logs/log.go

```diff
@@ -45,7 +45,8 @@ import (
 
 // RFC5424 log message levels.
 const (
-	LevelEmergency = iota
+	LevelCustom = iota //level for custom
```

Comment:
Users can customize the level of log;
For example,
the level of business log: [INFO][DEBUG][ERROR]....  
the level of database log:[SQL] [PARAMS] [RESULT] [INSERT] [SELECT] [UPDATE]...
their can be different.